src/apps/companies/apps/exports/group-history-items.js

```diff
@@ -0,0 +1,45 @@
+const GROUP_TIME_DELTA = 1000
+
+const isInteraction = (item) => item.hasOwnProperty('kind')
+
+function createCountries(item) {
```

Comment:
Optional would it be better not to modify the original value 
```suggestion
function createCountries(item) {
  const { country, ...rest } = item
  return {
    ...rest,
    countries: country
  }
  
}
```

Reply:
I have refactored to remove the function, thanks 👍